readme.txt

```diff
@@ -1,8 +1,8 @@
 === The Events Calendar ===
```

Comment:
The changes to the readme are not in scope with the issue linked in the description.
Pull them out of this PR and PR them separately, specifying the issue they are related to.
Or update the description to link to the issue the changes are for.

Reply:
I think this was pulled from Master. Made by Z.